typescript/src/proof.ts

```diff
@@ -38,7 +47,7 @@ export async function assembleTransactionProof(
 
   const headersChain = await bitcoinClient.getHeadersChain(
     txBlockHeight,
-    requiredConfirmations
+    requiredConfirmations - 1
```

Comment:
Worth clarifying why we need to subtract one here.

Reply:
Done in 3488910e67c7c1147fbf8ee0cb34a2c23bd269b9.